lib/IRGen/IRGenModule.cpp

```diff
@@ -367,17 +367,39 @@ IRGenModule::IRGenModule(IRGenerator &irgen,
   // TODO: Check that the deployment target supports the new
   // calling convention. Older versions of the runtime library
   // may not contain the entries using the new calling convention.
-
+  //
   // Only use the new calling conventions on platforms that support it.
+  //
+  // NB: Runtime is built with target clang, swiftc (this file) is
```

Comment:
In the context of this patch, I am referring to the just-built clang when I say "target clang", and the build-system's clang when I say "host clang". I'd be happy to rephrase that comment if it's unclear (given that clang is always a cross-compiler, the older senses of the terms "host compiler" or "target compiler" might not be worth using) but .. there's not a third compiler I'm accidentally being ambiguous about here, is there? There are only two clangs right? The one in /usr/bin and the one we just built.